LykkeApi2/Controllers/AssetsController.cs

```diff
@@ -228,10 +228,12 @@ public async Task<IActionResult> GetBaseAsset()
         [ProducesResponseType((int) HttpStatusCode.NotFound)]
         public async Task<IActionResult> SetBaseAsset([FromBody] BaseAssetUpdateModel model)
         {
-            if (string.IsNullOrWhiteSpace(model.BaseAsssetId))
+            var baseAssetFromRequest = model.BaseAssetId ?? model.BaseAsssetId;

```

Comment:
Why do we need a new variable here at all? `vaseAssetFromRequest` seems to be not more than alias for `model.BaseAssetId` - no preprocessing is performed, and it's used literally the same way. Or don't I get it right?

In other words, if this is a fix, so where do we have a difference in logic?

Reply:
in case BaseAssetId is filled, we use it, if it's not, we check BaseAsssetId - it's done for temporary backwards compatibility.